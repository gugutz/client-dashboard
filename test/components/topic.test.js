import React from 'react'
import renderer from 'react-test-renderer'
import Topic from '../../src/components/topic'

describe('<Topic /> component', () => {
  it('should render correctly', () => {
    const match = {
      params: {
        topicId: 'somerandomid'
      }
    }

    const tree = renderer.create(<Topic match={match}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
