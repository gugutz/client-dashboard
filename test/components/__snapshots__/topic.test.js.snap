// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Topic /> component should render correctly 1`] = `
<div>
  <h2
    className="subtitle"
  >
    somerandomid
  </h2>
</div>
`;
